website/cue/reference/components/sinks/vector.cue

```diff
@@ -83,6 +83,12 @@ components: sinks: vector: {
 				examples: ["92.12.333.224:\(_port)"]
 			}
 		}
+		compression: {
+			description: "Enable gRPC compression"
```

Comment:
I would make it clear that the compression method is `gzip` (for now).

Reply:
Done